2023-02-18 12:46:35,105 INFO    StreamThr :2961 [internal.py:wandb_internal():87] W&B internal server running at pid: 2961, started at: 2023-02-18 12:46:35.104974
2023-02-18 12:46:35,107 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status
2023-02-18 12:46:35,107 INFO    WriterThread:2961 [datastore.py:open_for_write():85] open: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/run-na9iyscw.wandb
2023-02-18 12:46:35,108 DEBUG   SenderThread:2961 [sender.py:send():336] send: header
2023-02-18 12:46:35,112 DEBUG   SenderThread:2961 [sender.py:send():336] send: run
2023-02-18 12:46:35,867 INFO    SenderThread:2961 [dir_watcher.py:__init__():216] watching files in: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files
2023-02-18 12:46:35,867 INFO    SenderThread:2961 [sender.py:_start_run_threads():1067] run started: na9iyscw with start time 1676745995.104171
2023-02-18 12:46:35,867 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: summary_record
2023-02-18 12:46:35,867 INFO    SenderThread:2961 [sender.py:_save_file():1321] saving file wandb-summary.json with policy end
2023-02-18 12:46:35,868 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: check_version
2023-02-18 12:46:35,868 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: check_version
2023-02-18 12:46:36,259 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: run_start
2023-02-18 12:46:36,328 DEBUG   HandlerThread:2961 [system_info.py:__init__():31] System info init
2023-02-18 12:46:36,329 DEBUG   HandlerThread:2961 [system_info.py:__init__():46] System info init done
2023-02-18 12:46:36,329 INFO    HandlerThread:2961 [system_monitor.py:start():151] Starting system monitor
2023-02-18 12:46:36,329 INFO    SystemMonitor:2961 [system_monitor.py:_start():116] Starting system asset monitoring threads
2023-02-18 12:46:36,329 INFO    HandlerThread:2961 [system_monitor.py:probe():172] Collecting system info
2023-02-18 12:46:36,329 INFO    SystemMonitor:2961 [interfaces.py:start():168] Started cpu
2023-02-18 12:46:36,330 INFO    SystemMonitor:2961 [interfaces.py:start():168] Started disk
2023-02-18 12:46:36,330 INFO    SystemMonitor:2961 [interfaces.py:start():168] Started gpu
2023-02-18 12:46:36,331 INFO    SystemMonitor:2961 [interfaces.py:start():168] Started memory
2023-02-18 12:46:36,332 INFO    SystemMonitor:2961 [interfaces.py:start():168] Started network
2023-02-18 12:46:36,353 DEBUG   HandlerThread:2961 [system_info.py:probe():195] Probing system
2023-02-18 12:46:36,408 DEBUG   HandlerThread:2961 [system_info.py:_probe_git():180] Probing git
2023-02-18 12:46:36,414 DEBUG   HandlerThread:2961 [system_info.py:_probe_git():188] Probing git done
2023-02-18 12:46:36,414 DEBUG   HandlerThread:2961 [system_info.py:probe():241] Probing system done
2023-02-18 12:46:36,414 DEBUG   HandlerThread:2961 [system_monitor.py:probe():181] {'os': 'Linux-5.10.16.3-microsoft-standard-WSL2-x86_64-with-glibc2.17', 'python': '3.8.16', 'heartbeatAt': '2023-02-18T18:46:36.353315', 'startedAt': '2023-02-18T18:46:35.100520', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': 'main.py', 'codePath': 'main.py', 'git': {'remote': 'https://github.com/jeffreymo572/Kaggles.git', 'commit': 'a0cd8bb23fb0060ab4fe283ffb1610fc09f1c40c'}, 'email': 'jeffreymo572@gmail.com', 'root': '/home/jeffreymo572/Kaggles', 'host': 'DESKTOP-NUCQ9A9', 'username': 'jeffreymo572', 'executable': '/home/jeffreymo572/anaconda3/envs/Kaggles/bin/python', 'cpu_count': 8, 'cpu_count_logical': 16, 'cpu_freq': {'current': 3600.0120000000006, 'min': 0.0, 'max': 0.0}, 'cpu_freq_per_core': [{'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}, {'current': 3600.012, 'min': 0.0, 'max': 0.0}], 'disk': {'total': 250.98248672485352, 'used': 144.22605514526367}, 'gpu': 'NVIDIA GeForce RTX 2080 Ti', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 2080 Ti', 'memory_total': 11811160064}], 'memory': {'total': 7.737171173095703}}
2023-02-18 12:46:36,415 INFO    HandlerThread:2961 [system_monitor.py:probe():182] Finished collecting system info
2023-02-18 12:46:36,415 INFO    HandlerThread:2961 [system_monitor.py:probe():185] Publishing system info
2023-02-18 12:46:36,415 DEBUG   HandlerThread:2961 [system_info.py:_save_pip():51] Saving list of pip packages installed into the current environment
2023-02-18 12:46:36,415 DEBUG   HandlerThread:2961 [system_info.py:_save_pip():67] Saving pip packages done
2023-02-18 12:46:36,415 DEBUG   HandlerThread:2961 [system_info.py:_save_conda():74] Saving list of conda packages installed into the current environment
2023-02-18 12:46:36,868 INFO    Thread-13 :2961 [dir_watcher.py:_on_file_created():275] file/dir created: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/wandb-summary.json
2023-02-18 12:46:36,868 INFO    Thread-13 :2961 [dir_watcher.py:_on_file_created():275] file/dir created: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/conda-environment.yaml
2023-02-18 12:46:36,868 INFO    Thread-13 :2961 [dir_watcher.py:_on_file_created():275] file/dir created: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/requirements.txt
2023-02-18 12:46:42,436 DEBUG   HandlerThread:2961 [system_info.py:_save_conda():86] Saving conda packages done
2023-02-18 12:46:42,437 INFO    HandlerThread:2961 [system_monitor.py:probe():187] Finished publishing system info
2023-02-18 12:46:42,440 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:46:42,440 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:46:42,441 DEBUG   SenderThread:2961 [sender.py:send():336] send: files
2023-02-18 12:46:42,441 INFO    SenderThread:2961 [sender.py:_save_file():1321] saving file wandb-metadata.json with policy now
2023-02-18 12:46:42,445 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: stop_status
2023-02-18 12:46:42,446 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: stop_status
2023-02-18 12:46:42,805 DEBUG   SenderThread:2961 [sender.py:send():336] send: telemetry
2023-02-18 12:46:42,869 INFO    Thread-13 :2961 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/conda-environment.yaml
2023-02-18 12:46:42,869 INFO    Thread-13 :2961 [dir_watcher.py:_on_file_created():275] file/dir created: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/wandb-metadata.json
2023-02-18 12:46:43,382 INFO    wandb-upload_0:2961 [upload_job.py:push():138] Uploaded file /tmp/tmpjvvl2lmjwandb/7bvqts0s-wandb-metadata.json
2023-02-18 12:46:45,805 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:46:50,806 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:46:55,807 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:46:57,445 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:46:57,445 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:00,807 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:02,446 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:05,810 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:06,874 INFO    Thread-13 :2961 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/config.yaml
2023-02-18 12:47:07,446 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:11,221 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:12,447 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:16,222 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:17,449 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:21,222 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:22,450 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:26,223 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:27,451 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:31,224 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:32,451 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:36,225 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:36,332 DEBUG   SystemMonitor:2961 [system_monitor.py:_start():130] Starting system metrics aggregation loop
2023-02-18 12:47:36,333 DEBUG   SenderThread:2961 [sender.py:send():336] send: stats
2023-02-18 12:47:37,452 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:41,334 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:42,453 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:46,335 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:47,454 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:51,335 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:52,455 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:47:56,336 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:47:57,456 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:01,337 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:02,458 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:06,333 DEBUG   SenderThread:2961 [sender.py:send():336] send: stats
2023-02-18 12:48:07,334 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:07,459 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:10,363 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: partial_history
2023-02-18 12:48:10,363 DEBUG   SenderThread:2961 [sender.py:send():336] send: history
2023-02-18 12:48:10,364 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: summary_record
2023-02-18 12:48:10,364 INFO    SenderThread:2961 [sender.py:_save_file():1321] saving file wandb-summary.json with policy end
2023-02-18 12:48:10,888 INFO    Thread-13 :2961 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/wandb-summary.json
2023-02-18 12:48:12,364 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:12,460 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:17,365 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:17,462 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:22,366 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:22,463 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:27,367 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:27,464 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:32,367 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:32,465 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:36,334 DEBUG   SenderThread:2961 [sender.py:send():336] send: stats
2023-02-18 12:48:37,466 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:38,335 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:42,466 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:43,336 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:47,467 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:48,336 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:52,468 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:53,337 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:48:57,469 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:48:58,338 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:49:02,470 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:49:03,338 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:49:06,334 DEBUG   SenderThread:2961 [sender.py:send():336] send: stats
2023-02-18 12:49:07,471 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: keepalive
2023-02-18 12:49:09,335 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:49:12,110 DEBUG   SenderThread:2961 [sender.py:send():336] send: exit
2023-02-18 12:49:12,110 INFO    SenderThread:2961 [sender.py:send_exit():559] handling exit code: 255
2023-02-18 12:49:12,110 INFO    SenderThread:2961 [sender.py:send_exit():561] handling runtime: 155
2023-02-18 12:49:12,110 INFO    SenderThread:2961 [sender.py:_save_file():1321] saving file wandb-summary.json with policy end
2023-02-18 12:49:12,110 INFO    SenderThread:2961 [sender.py:send_exit():567] send defer
2023-02-18 12:49:12,110 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: defer
2023-02-18 12:49:12,110 INFO    HandlerThread:2961 [handler.py:handle_request_defer():170] handle defer: 0
2023-02-18 12:49:12,111 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: defer
2023-02-18 12:49:12,111 INFO    SenderThread:2961 [sender.py:send_request_defer():583] handle sender defer: 0
2023-02-18 12:49:12,111 INFO    SenderThread:2961 [sender.py:transition_state():587] send defer: 1
2023-02-18 12:49:12,111 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: defer
2023-02-18 12:49:12,111 INFO    HandlerThread:2961 [handler.py:handle_request_defer():170] handle defer: 1
2023-02-18 12:49:12,111 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: defer
2023-02-18 12:49:12,111 INFO    SenderThread:2961 [sender.py:send_request_defer():583] handle sender defer: 1
2023-02-18 12:49:12,111 INFO    SenderThread:2961 [sender.py:transition_state():587] send defer: 2
2023-02-18 12:49:12,111 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: defer
2023-02-18 12:49:12,111 INFO    HandlerThread:2961 [handler.py:handle_request_defer():170] handle defer: 2
2023-02-18 12:49:12,111 INFO    HandlerThread:2961 [system_monitor.py:finish():161] Stopping system monitor
2023-02-18 12:49:12,112 DEBUG   SystemMonitor:2961 [system_monitor.py:_start():137] Finished system metrics aggregation loop
2023-02-18 12:49:12,112 DEBUG   SystemMonitor:2961 [system_monitor.py:_start():141] Publishing last batch of metrics
2023-02-18 12:49:12,113 INFO    HandlerThread:2961 [interfaces.py:finish():175] Joined cpu
2023-02-18 12:49:12,113 INFO    HandlerThread:2961 [interfaces.py:finish():175] Joined disk
2023-02-18 12:49:12,118 INFO    HandlerThread:2961 [interfaces.py:finish():175] Joined gpu
2023-02-18 12:49:12,118 INFO    HandlerThread:2961 [interfaces.py:finish():175] Joined memory
2023-02-18 12:49:12,118 INFO    HandlerThread:2961 [interfaces.py:finish():175] Joined network
2023-02-18 12:49:12,119 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: defer
2023-02-18 12:49:12,119 INFO    SenderThread:2961 [sender.py:send_request_defer():583] handle sender defer: 2
2023-02-18 12:49:12,119 INFO    SenderThread:2961 [sender.py:transition_state():587] send defer: 3
2023-02-18 12:49:12,119 DEBUG   SenderThread:2961 [sender.py:send():336] send: stats
2023-02-18 12:49:12,119 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: defer
2023-02-18 12:49:12,119 INFO    HandlerThread:2961 [handler.py:handle_request_defer():170] handle defer: 3
2023-02-18 12:49:12,120 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: defer
2023-02-18 12:49:12,120 INFO    SenderThread:2961 [sender.py:send_request_defer():583] handle sender defer: 3
2023-02-18 12:49:12,120 INFO    SenderThread:2961 [sender.py:transition_state():587] send defer: 4
2023-02-18 12:49:12,120 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: defer
2023-02-18 12:49:12,120 INFO    HandlerThread:2961 [handler.py:handle_request_defer():170] handle defer: 4
2023-02-18 12:49:12,120 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: defer
2023-02-18 12:49:12,120 INFO    SenderThread:2961 [sender.py:send_request_defer():583] handle sender defer: 4
2023-02-18 12:49:12,120 INFO    SenderThread:2961 [sender.py:transition_state():587] send defer: 5
2023-02-18 12:49:12,120 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: defer
2023-02-18 12:49:12,120 INFO    HandlerThread:2961 [handler.py:handle_request_defer():170] handle defer: 5
2023-02-18 12:49:12,120 DEBUG   SenderThread:2961 [sender.py:send():336] send: summary
2023-02-18 12:49:12,121 INFO    SenderThread:2961 [sender.py:_save_file():1321] saving file wandb-summary.json with policy end
2023-02-18 12:49:12,121 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: defer
2023-02-18 12:49:12,121 INFO    SenderThread:2961 [sender.py:send_request_defer():583] handle sender defer: 5
2023-02-18 12:49:12,121 INFO    SenderThread:2961 [sender.py:transition_state():587] send defer: 6
2023-02-18 12:49:12,121 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: defer
2023-02-18 12:49:12,121 INFO    HandlerThread:2961 [handler.py:handle_request_defer():170] handle defer: 6
2023-02-18 12:49:12,121 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: defer
2023-02-18 12:49:12,121 INFO    SenderThread:2961 [sender.py:send_request_defer():583] handle sender defer: 6
2023-02-18 12:49:12,121 INFO    SenderThread:2961 [sender.py:transition_state():587] send defer: 7
2023-02-18 12:49:12,121 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: status_report
2023-02-18 12:49:12,121 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: defer
2023-02-18 12:49:12,121 INFO    HandlerThread:2961 [handler.py:handle_request_defer():170] handle defer: 7
2023-02-18 12:49:12,122 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: defer
2023-02-18 12:49:12,122 INFO    SenderThread:2961 [sender.py:send_request_defer():583] handle sender defer: 7
2023-02-18 12:49:12,901 INFO    Thread-13 :2961 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/wandb-summary.json
2023-02-18 12:49:12,902 INFO    Thread-13 :2961 [dir_watcher.py:_on_file_created():275] file/dir created: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/output.log
2023-02-18 12:49:13,109 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: poll_exit
2023-02-18 12:49:14,113 INFO    SenderThread:2961 [sender.py:transition_state():587] send defer: 8
2023-02-18 12:49:14,113 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: poll_exit
2023-02-18 12:49:14,113 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: defer
2023-02-18 12:49:14,113 INFO    HandlerThread:2961 [handler.py:handle_request_defer():170] handle defer: 8
2023-02-18 12:49:14,113 DEBUG   SenderThread:2961 [sender.py:send_request():363] send_request: defer
2023-02-18 12:49:14,113 INFO    SenderThread:2961 [sender.py:send_request_defer():583] handle sender defer: 8
2023-02-18 12:49:14,119 INFO    SenderThread:2961 [sender.py:transition_state():587] send defer: 9
2023-02-18 12:49:14,120 DEBUG   HandlerThread:2961 [handler.py:handle_request():144] handle_request: defer
2023-02-18 12:49:14,120 INFO    HandlerThread:2961 [handler.py:handle_request_defer():170] handle defer: 9
2023-02-18 12:49:14,120 DEBUG   SenderThread:2961 [sender.py:send():336] send: artifact
2023-02-18 12:49:14,902 INFO    Thread-13 :2961 [dir_watcher.py:_on_file_modified():292] file/dir modified: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/output.log
2023-02-18 12:49:16,251 INFO    wandb-upload_1:2961 [upload_job.py:push():93] Skipped uploading /home/jeffreymo572/.local/share/wandb/artifacts/staging/tmp0uq59i4g
2023-02-18 12:49:16,276 WARNING StreamThr :2961 [internal.py:is_dead():415] Internal process exiting, parent pid 2916 disappeared
2023-02-18 12:49:16,276 ERROR   StreamThr :2961 [internal.py:wandb_internal():153] Internal process shutdown.
2023-02-18 12:49:16,748 INFO    wandb-upload_0:2961 [upload_job.py:push():96] Uploaded file /home/jeffreymo572/.local/share/wandb/artifacts/staging/tmp25q04dxj
2023-02-18 12:49:17,120 INFO    HandlerThread:2961 [handler.py:finish():841] shutting down handler
2023-02-18 12:49:17,120 INFO    WriterThread:2961 [datastore.py:close():299] close: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/run-na9iyscw.wandb
2023-02-18 12:49:18,912 INFO    SenderThread:2961 [sender.py:send_artifact():1417] sent artifact job-https___github.com_jeffreymo572_Kaggles.git_main.py - {'id': 'QXJ0aWZhY3Q6MzczNDcwNDA4', 'digest': '571be68f4496dbeda109f39d62c0667a', 'state': 'PENDING', 'aliases': [], 'artifactSequence': {'id': 'QXJ0aWZhY3RDb2xsZWN0aW9uOjUzNDY5NTQ4', 'latestArtifact': {'id': 'QXJ0aWZhY3Q6MzczNDQxOTk2', 'versionIndex': 2}}, 'version': 'latest'}
2023-02-18 12:49:18,912 INFO    SenderThread:2961 [sender.py:finish():1493] shutting down sender
2023-02-18 12:49:18,912 INFO    SenderThread:2961 [dir_watcher.py:finish():362] shutting down directory watcher
2023-02-18 12:49:19,903 INFO    SenderThread:2961 [dir_watcher.py:finish():392] scan: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files
2023-02-18 12:49:19,903 INFO    SenderThread:2961 [dir_watcher.py:finish():406] scan save: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/output.log output.log
2023-02-18 12:49:19,903 INFO    SenderThread:2961 [dir_watcher.py:finish():406] scan save: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/conda-environment.yaml conda-environment.yaml
2023-02-18 12:49:19,904 INFO    SenderThread:2961 [dir_watcher.py:finish():406] scan save: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/config.yaml config.yaml
2023-02-18 12:49:19,905 INFO    SenderThread:2961 [dir_watcher.py:finish():406] scan save: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/wandb-metadata.json wandb-metadata.json
2023-02-18 12:49:19,905 INFO    SenderThread:2961 [dir_watcher.py:finish():406] scan save: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/requirements.txt requirements.txt
2023-02-18 12:49:19,906 INFO    SenderThread:2961 [dir_watcher.py:finish():406] scan save: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/wandb-summary.json wandb-summary.json
2023-02-18 12:49:19,906 INFO    SenderThread:2961 [file_pusher.py:finish():162] shutting down file pusher
2023-02-18 12:49:19,906 INFO    SenderThread:2961 [file_pusher.py:join():167] waiting for file pusher
2023-02-18 12:49:20,266 ERROR   Thread-12 :2961 [step_upload.py:_handle_event():141] Failed to upload file: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/output.log
Traceback (most recent call last):
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 26, in wrapper
    return func(*args, **kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1769, in upload_urls
    raise CommError(f"Run does not exist {entity}/{project}/{run_id}.")
wandb.errors.CommError: Run does not exist jeffreymo/KaggleS3E7/na9iyscw.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/step_upload.py", line 235, in run_and_notify
    job.run()
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/upload_job.py", line 62, in run
    self.push()
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/upload_job.py", line 113, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 64, in wrapper
    raise CommError(message, err).with_traceback(sys.exc_info()[2])
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 26, in wrapper
    return func(*args, **kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1769, in upload_urls
    raise CommError(f"Run does not exist {entity}/{project}/{run_id}.")
wandb.errors.CommError: Run does not exist jeffreymo/KaggleS3E7/na9iyscw.
2023-02-18 12:49:20,273 ERROR   Thread-12 :2961 [step_upload.py:_handle_event():141] Failed to upload file: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/requirements.txt
Traceback (most recent call last):
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 26, in wrapper
    return func(*args, **kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1769, in upload_urls
    raise CommError(f"Run does not exist {entity}/{project}/{run_id}.")
wandb.errors.CommError: Run does not exist jeffreymo/KaggleS3E7/na9iyscw.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/step_upload.py", line 235, in run_and_notify
    job.run()
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/upload_job.py", line 62, in run
    self.push()
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/upload_job.py", line 113, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 64, in wrapper
    raise CommError(message, err).with_traceback(sys.exc_info()[2])
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 26, in wrapper
    return func(*args, **kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1769, in upload_urls
    raise CommError(f"Run does not exist {entity}/{project}/{run_id}.")
wandb.errors.CommError: Run does not exist jeffreymo/KaggleS3E7/na9iyscw.
2023-02-18 12:49:20,275 ERROR   Thread-12 :2961 [step_upload.py:_handle_event():141] Failed to upload file: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/conda-environment.yaml
Traceback (most recent call last):
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 26, in wrapper
    return func(*args, **kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1769, in upload_urls
    raise CommError(f"Run does not exist {entity}/{project}/{run_id}.")
wandb.errors.CommError: Run does not exist jeffreymo/KaggleS3E7/na9iyscw.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/step_upload.py", line 235, in run_and_notify
    job.run()
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/upload_job.py", line 62, in run
    self.push()
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/upload_job.py", line 113, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 64, in wrapper
    raise CommError(message, err).with_traceback(sys.exc_info()[2])
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 26, in wrapper
    return func(*args, **kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1769, in upload_urls
    raise CommError(f"Run does not exist {entity}/{project}/{run_id}.")
wandb.errors.CommError: Run does not exist jeffreymo/KaggleS3E7/na9iyscw.
2023-02-18 12:49:20,277 ERROR   Thread-12 :2961 [step_upload.py:_handle_event():141] Failed to upload file: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/config.yaml
Traceback (most recent call last):
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 26, in wrapper
    return func(*args, **kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1769, in upload_urls
    raise CommError(f"Run does not exist {entity}/{project}/{run_id}.")
wandb.errors.CommError: Run does not exist jeffreymo/KaggleS3E7/na9iyscw.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/step_upload.py", line 235, in run_and_notify
    job.run()
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/upload_job.py", line 62, in run
    self.push()
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/upload_job.py", line 113, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 64, in wrapper
    raise CommError(message, err).with_traceback(sys.exc_info()[2])
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 26, in wrapper
    return func(*args, **kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1769, in upload_urls
    raise CommError(f"Run does not exist {entity}/{project}/{run_id}.")
wandb.errors.CommError: Run does not exist jeffreymo/KaggleS3E7/na9iyscw.
2023-02-18 12:49:20,282 ERROR   Thread-12 :2961 [step_upload.py:_handle_event():141] Failed to upload file: /home/jeffreymo572/Kaggles/wandb/run-20230218_124635-na9iyscw/files/wandb-summary.json
Traceback (most recent call last):
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 26, in wrapper
    return func(*args, **kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1769, in upload_urls
    raise CommError(f"Run does not exist {entity}/{project}/{run_id}.")
wandb.errors.CommError: Run does not exist jeffreymo/KaggleS3E7/na9iyscw.

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/step_upload.py", line 235, in run_and_notify
    job.run()
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/upload_job.py", line 62, in run
    self.push()
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/filesync/upload_job.py", line 113, in push
    _, upload_headers, result = self._api.upload_urls(project, [self.save_name])
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 64, in wrapper
    raise CommError(message, err).with_traceback(sys.exc_info()[2])
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/apis/normalize.py", line 26, in wrapper
    return func(*args, **kwargs)
  File "/home/jeffreymo572/anaconda3/envs/Kaggles/lib/python3.8/site-packages/wandb/sdk/internal/internal_api.py", line 1769, in upload_urls
    raise CommError(f"Run does not exist {entity}/{project}/{run_id}.")
wandb.errors.CommError: Run does not exist jeffreymo/KaggleS3E7/na9iyscw.
